pkg/virt-handler/isolation/process_test.go

```diff
@@ -0,0 +1,134 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package isolation
+
+import (
+	"github.com/mitchellh/go-ps"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+var emptyProcessList = []ps.Process{}
+
+var _ = Describe("process", func() {
+	const proc1Andproc5Exec = "processA"
+	const proc2Andproc4Pid = 110
+	const nonExistPPid = 300
+
+	procStub1 := ProcessStub{ppid: 1, pid: 120, binary: proc1Andproc5Exec}
+	procStub2 := ProcessStub{ppid: proc2Andproc4Pid, pid: 2222, binary: "processB"}
+	procStub3 := ProcessStub{ppid: 1, pid: 110, binary: "processC"}
+	procStub4 := ProcessStub{ppid: proc2Andproc4Pid, pid: 3333, binary: "processD"}
+	procsStubList := []ProcessStub{procStub1, procStub3, procStub2, procStub4}
+	processes := convertToPsGoProcessList(procsStubList)
+
+	Context("find child processes", func() {
+		proc2AndProc4StubList := []ProcessStub{procStub2, procStub4}
+		proc2AndProc4 := convertToPsGoProcessList(proc2AndProc4StubList)
```

Comment:
I just wanted this to be done in the test function so we will have a simple function signature and do the conversions at the place it is needed, not before.

Now we have a global (`emptyProcessList`) and these two variables in the context container which are visible to all tests under it.
The simple path, is also the better choice here. Please move the conversion in the function and remove all these 3 redundant variables.

Reply:
I removed `proc2AndProc4StubList` and `proc2AndProc4`, 
now their values are inline.
Moved `emptyProcessList` to the describe block,
And moved conversions calls to the test function.

Having `emptyProcessList` is more readable, isnt it?